converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -31,42 +64,26 @@ public Detail() {
 	 * Copy constructor
 	 */
 	public Detail(Detail detail) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1213)